articles/active-directory-b2c/force-password-reset.md

```diff
@@ -132,6 +133,8 @@ Content-type: application/json
 
 If you disabled the strong [password complexity](password-complexity.md), update the password policy to [DisableStrongPassword](user-profile-attributes.md#password-policy-attribute):
 
+Note: After a user reset their password, the passwordPolicies will be changed back to DisablePasswordExpiration
```

Comment:
```suggestion
> [!NOTE]
> After the user resets their password, the passwordPolicies will be changed back to DisablePasswordExpiration
```

Reply:
@tusharka1 See my suggested commit.

Reply:
Looks good @kengaderdus

Reply:
@tusharka1 

You can then commit those changes as I can't do it from my end.

Reply:
@tusharka1 

Looks like you've not seen my suggested commit. Please check it out and address it so that we can sign off your RP.